hpcgap/lib/oper.g

```diff
@@ -1276,6 +1276,106 @@ BIND_GLOBAL( "DeclareAttribute", function ( arg )
 end );
 
 
+##############################################################################
+##
+##  This code adds thread-local attributes, which are used in HPC-GAP.
+##  These are not complete replacements for normal attributes.
+##  In particular, they are not filters, so cannot be used in method selection.
+##
+##  The main problem with thread-local attributes is that only a subset of the
```

Comment:
This paragraph does not mesh well with the preceding one, it reads as if it was written at a different time, with disregard for what came before.
So perhaps instead start like so: "To elaborate, the main problem [...]" and then remove the last sentence starting with "Therefore". And perhaps meld this paragraph (now reduced to a single sentence) with the preceding paragraph.